vcpkg/maintainers/registries.md

```diff
@@ -1,281 +1,289 @@
 
-- The version of the port in question; should be exactly the same as the `vcpkg.json` file, including the version fields and `"port-version"`.
-- The `"path"` field: a relative directory, rooted at the base of the registry (in other words, the directory where `versions` is located), to the port directory. It should look something like `"$/path/to/port/dir`"
+| Name       | Type            | Description |
+| ---------- | --------------- | --------------------------- |
+| `versions` | VersionObject[] | An array of version objects. Contains an entry for each version of the port in the registry. |
 
-The version field for ports with deprecated `CONTROL` files is `"version-string"`.
+*VersionObject*
 
-In general, it's not important where you place port directories. However, the idiom in vcpkg is to follow somewhat closely to what the built in vcpkg registry does: your `kitten` port at version `x.y.z` should be placed in `ports/kitten/x.y.z`, with port versions appended as you see fit (although since `#` is not a good character to use for file names, perhaps use `_`).
+| Name       | Type   | Description |
+| ---------- | ------ | ----------- |
+| `path`     | string | The filesystem location where the port files for the corresponding version are located |
+| [`version`<br>`version-semver`<br>`version-date`<br>`version-string`](../reference/vcpkg-json.md#version) | string | Upstream version information |
+| [port-version](../reference/vcpkg-json.md#port-version) | integer | Port files revision |
 
-> [!WARNING]
-> One very important part of registries is that versions should _never_ be changed. One should never remove or change an existing version. Your changes to your registry shouldn't change behavior to downstream users.
+When specifying the `path` of a registry, the `$` character can be used to
+reference the root of the registry. Otherwise, absolute paths can be used
+instead.
 
-Here's an example of a valid version database for a `kitten` port with one version:
+#### Example of a filesystem registry version file
 
 ```json
 {
   "versions": [
     {
-      "version": "2.6.2",
-      "port-version": 0,
-      "path": "$/ports/kitten/2.6.2_0"
-    }
-  ]
-}
-```
-
-#### Add a new version
-
-Unlike git registries, adding a new version to a filesystem registry mostly involves a lot of copying. The first thing to do is to copy the latest version of your port into a new version directory, update the version and `"port-version"` fields as you need to, and then test with `overlay-ports`:
-
-  `vcpkg install kitten --overlay-ports=ports/kitten/new-version`.
-
-Once you've finished your testing, you can add this new version to the top of your `versions/k-/kitten.json`:
-
-```json
-{
-  "versions": [
+      "git-tree": "$/ports/foo/1.2.0",
+      "version": "1.2.0",
+      "port-version": 0
+    },
     {
-      "version": "2.6.3",
-      "port-version": 0,
-      "path": "$/ports/kitten/2.6.3_0"
+      "git-tree": "$/ports/foo/1.1.0",
```

Comment:
This should be `"path"` https://github.com/microsoft/vcpkg-tool/blob/b8680ed15eb1350e420a18ce37556d3dc9e6d084/src/vcpkg/registries.cpp#L117